### DISCUSSION TITLE
Simulating long positions

### DISCUSSION THREAD

**Discussion by aprodsilva** [vsaveris/trading-technical-indicators#37] [Q&A] [answered]
Hi, I am trying this great library and wondering: 
Would it possible to simulate the trading only using long positions?
Thanks for your attention.

**Comment by vsaveris** [comment, marked as answer]
Hi!

Not for the moment, but this is a good idea. 

A next release is planned for end of February, I can include this feature in this version. 

If you have more thoughts on how this simulation should work, you can draft your ideas and open a ticket in github.

Regards!

On 21 Dec 2021, at 21:37, aprodsilva ***@***.***> wrote:

﻿
Hi, I am trying this great library and wondering:
Would it possible to simulate the trading only using long positions?
Thanks for your attention.
